synapse/lib/node.py

```diff
@@ -39,6 +39,9 @@ def __init__(self, snap, buid=None, rawprops=None):
         if self.ndef is not None:
             self.form = self.snap.model.form(self.ndef[0])
 
+    def __repr__(self):
```

Comment:
this is uncovered in test code?

Reply:
was this supposed to be in the Node class of hive.py?

Reply:
no